examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        idata1 = pm.sample_prior_predictive(samples=N)
</pre><p>it would be nice to pass the <code>rng</code> to all the sample stems in the notebook to make it fully reproducible. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>good idea, I missed this. Fixed</p>